WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -58,7 +58,7 @@ class ReaderDiscoverViewModel @Inject constructor(
     private val photonWidth: Int = 500
     private val photonHeight: Int = 500
 
-    private lateinit var readerDiscoverRepository: ReaderDiscoverRepository
+    private lateinit var mReaderDiscoverDataProvider: ReaderDiscoverDataProvider
```

Comment:
Hi @zwarm for kotlin files, I don't think the member` m` in front of the variable is used. Let me know your thoughts on this if there's another reason it was used. Thank you 🙏

Reply:
Yes, you are correct. That was AndroidStudio auto rename/refactor. Nice catch.

Reply:
Replaced in c39c765